src/test/java/org/spectrumauctions/sats/mechanism/cca/GSVMCCATest.java

```diff
@@ -82,17 +80,37 @@ private void testClockPhaseVsSupplementaryPhaseEfficiency() {
 
         assertTrue(qualityCP.compareTo(qualitySR) < 1);
         assertTrue(qualitySR.compareTo(BigDecimal.ONE) < 1);
+
+        logger.info("Calculating payments...");
+        Payment<GSVMLicense> payment = cca.getPayment();
+        logger.info("Done.");
+    }
+
+    @Test
+    public void testCCAWithCCG() {
+        long seed = 10059L;
+        while (true) {
+            List<GSVMBidder> rawBidders = new GlobalSynergyValueModel().createNewPopulation(seed++);
+            logger.info("Starting with seed {}.", seed);
+            NonGenericCCAMechanism<GSVMLicense> cca = getMechanism(rawBidders);
+            cca.calculateSampledStartingPrices(10, 50, 0.1, seed);
+            cca.setTimeLimit(20);
+            Payment<GSVMLicense> payment = cca.getPayment();
+            logger.info("Done.");
```

Comment:
This test confuses me. It can only exit the while loop in case of a runtime exception - in which case it fails. Right?
Also, 'payment' is never read...